### PR TITLE
docs: add component translations table to properties docs

### PR DESCRIPTION
**Pull request by langz**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/global-error/properties.mdx [hunk 1]
```diff
@@ -16,3 +18,7 @@ showTabs: true
 | `skeleton` | _(optional)_ applies a loading skeleton to all included elements.                                                                |
 | `locale`   | _(optional)_ set a [supported locale](/uilib/usage/customisation/localization/) if needed.                                       |
 | `children` | _(optional)_ to display additional related content, like useful links etc.                                                       |
+
+## Translations
+
+<TranslationsTable localeKey="GlobalError" />
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/global-error/properties.mdx:24]: Any ideas on how I can improve upon this one?
The GlobalError translation is a bit different structured than other translations, with nested objects:
```
    GlobalError: {
      404: {
        title: "We can't find the page you're looking for …",
        text: 'Are you sure you have entered the correct address? Or have we messed with the links?',
      },
      500: {
        title: 'Sorry, a technical error happened!',
        text: 'The service is not working properly at the moment. Try again later.',
      },
      code: 'Error code:',
      help: 'Here are some links that might help:',
    },
```

This now looks like this:
Screenshot 2024-09-05 at 21 13 26

Can also be viewed in the deploy preview.

Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/components/global-error/properties.mdx:24]: Try something like that:

```tsx
<TranslationsTable
  localeKey={[
    'GlobalError.404',
    'GlobalError.500',
    'GlobalError.code',
    'GlobalError.help',
  ]}
/>
```

Its not that nice, and needs custom support when they change. But we could make some magic inside the `TranslationsTable`, and check for nested object and destruct them. I think we already do it for one lever, do we?

**Review by tujoworker — APPROVED** [review]
Nice 🤩

File: packages/dnb-design-system-portal/src/docs/uilib/components/global-error/properties.mdx (hunk 1 above)
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/components/global-error/properties.mdx:24]: Thanks for the suggestion, that resulted in the same result.

I quickly tried to do some magic inside TranslationsTable, not the best code, but it does the job.

Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/components/global-error/properties.mdx:24]: Now looking like:

image

**Pull request merged**